ads/google/a4a/traffic-experiments.js

```diff
@@ -209,21 +208,13 @@ function selectRandomProperty(obj) {
  */
 export function randomlySelectUnsetPageExperiments(win, experiments) {
   win.pageExperimentBranches = win.pageExperimentBranches || {};
-  if (getMode(win).localDev) {
-    // In local dev mode, it can be difficult to configure AMP_CONFIG
-    // externally.  Default it here if necessary.
-    win.AMP_CONFIG = win.AMP_CONFIG || {};
```

Comment:
As explained below, we let `experiments.js` to handle all the experiment status read write. No one should access AMP_CONFIG directly. How we store the status in AMP_CONFIG is really an implementation detail, and subject to change.